lib/experimental/Navigation/Carousel/index.tsx

```diff
@@ -34,22 +35,23 @@ export const Carousel = ({
   delay = 3000,
   columns = { default: 1 },
   showPeek = false,
+  showFade = false,
 }: CarouselProps) => {
   const childrenArray = React.Children.toArray(children)
 
-  const plugin = React.useRef(
-    autoplay ? Autoplay({ delay: delay, stopOnInteraction: true }) : undefined
-  )
+  const plugin = useMemo(() => {
+    return autoplay ? Autoplay({ delay, stopOnInteraction: true }) : undefined
+  }, [autoplay, delay])
```

Comment:
We need this to reactivate autoplay if the prop autoplay change from `false` to `true`